cypress/e2e/smoke/dbot/DbotRedirections.cy.js

```diff
@@ -1,29 +1,54 @@
+import { derivApp } from '../../../support/locators'
+
 describe('QATEST-136582: Redirection to other pages from dbot', () => {
+  const size = ['small', 'desktop']
   beforeEach(() => {
     cy.c_login({ user: 'dBot', rateLimitCheck: true })
-    cy.c_visitResponsive('/appstore/traders-hub', 'large')
-    cy.c_openDbotThub()
-    cy.c_skipTour()
   })
 
-  it('Redirect to deposit page from Dbot', () => {
-    cy.findByRole('button', { name: 'Deposit' }).click()
-    cy.findByTestId('dt_acc_info').should('be.visible')
-    cy.findByText('Deposit via bank wire, credit card, and e-wallet').should(
-      'be.visible'
-    )
-  })
+  size.forEach((size) => {
+    it(`Redirect to deposit page from Dbot on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
+      const isMobile = size == 'small' ? true : false
+      cy.c_visitResponsive('appstore/traders-hub', size)
+      cy.findAllByTestId('dt_balance_text_container').should('have.length', '2')
+      if (isMobile) cy.c_skipPasskeysV2()
+      cy.c_openDbotThub()
+      cy.c_rateLimit()
+      if (isMobile) cy.findByTestId('close-icon', { timeout: 7000 }).click()
+      cy.c_skipTour()
+      if (isMobile) {
+        derivApp.commonPage.mobileLocators.header.hamburgerMenuButton().click()
+        cy.findByRole('heading', { name: 'Cashier' }).click()
+        cy.findByRole('link', { name: 'Deposit' }).click()
+      } else {
+        cy.findByRole('button', { name: 'Deposit' }).click()
+      }
+      cy.findByTestId('dt_acc_info').should('exist')
```

Comment:
assert on visibility rather than on existence.
```suggestion
      cy.findByTestId('dt_acc_info').should('be.visible')
```

Reply:
Getting failure in the test if using be.visible since the dt_acc_info is not visible due to it has CSS property of position:fixed and its being covered by another element @bilalmajid-deriv